libnavigation-ui/src/test/java/com/mapbox/navigation/ui/map/NavigationSymbolManagerTest.java

```diff
@@ -8,9 +8,7 @@
 import org.junit.Test;
 
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
do we also want to avoid the wildcard import in test code?

Reply:
Checkstyle doesn't check that while Ktlint does. I put the explicit imports back though 👍